src/rpc/budget.cpp

```diff
@@ -453,13 +453,13 @@ UniValue getbudgetprojection(const JSONRPCRequest& request)
             "    \"Yeas\": n,                    (numeric) Number of yea votes\n"
             "    \"Nays\": n,                    (numeric) Number of nay votes\n"
             "    \"Abstains\": n,                (numeric) Number of abstains\n"
-            "    \"TotalPayment\": xxx.xxx,      (numeric) Total payment amount\n"
-            "    \"MonthlyPayment\": xxx.xxx,    (numeric) Monthly payment amount\n"
-            "    \"IsEstablished\": true|false,  (boolean) Established (true) or (false)\n"
+            "    \"TotalPayment\": xxx.xxx,      (numeric) Total payment amount in PIV\n"
+            "    \"MonthlyPayment\": xxx.xxx,    (numeric) Monthly payment amount in PIV\n"
+            "    \"IsEstablished\": true|false,  (boolean) Established time is > 24 hours, (true) or (false)\n"
```

Comment:
A proposal becomes `established` when the block including the collateral is at least `nProposalEstablishmentTime` old (24 hours on mainnet, 5 minutes on testnet).

Reply:
would you prefer this exact text?

Reply:
Mhm... not sure what's the best wording, but maybe we should mention that time is not counted starting when the proposal is sent with `submitbudget`, but when the fee-collateral-tx is mined. 
We should probably also mention that the establishment time is different on testnet.

Also, no need to repeat `(true) or (false)` at the end (since this is a boolean argument, it's already clear imo).

Reply:
How about something like such:
`(boolean) Established transaction is older than 24 hours\n"`

Reply:
I think people don't realize that a Proposal is a 'transaction'. Therefore, that terminology may confuse them.
Maybe state;
`(boolean) Proposal is considered established, 24 hrs after being submitted to network.\n`
For testnet;
`(boolean) Proposal is considered established, 24 hrs after being submitted to network. (5 mins for Testnet)\n`